rllib/env/wrappers/open_spiel.py

```diff
@@ -9,7 +9,7 @@ class OpenSpielEnv(MultiAgentEnv):
     def __init__(self, env):
         super().__init__()
         self.env = env
-
+        self._skip_env_checking = True
```

Comment:
can you add a comment about what fails the checker for this env wrapper?
just so we have more context. thanks.

Reply:
It is the same as it is for all the environments -- if we don't implement the MultiAgentEnv space checking environments, then we are relying on the fact that the environment should be in the preferred MultiAgentEnv space format. 

I don't think its necessary to add it in because I'll probably be the one who has to bring the environments up to standard. But I think instead what would be useful is if I wrote a doc page explaining what are the requirements of implementing a good environment in RLlib, that way users know how to solve their problems without having to read the check_env function.

I can do that in a separate doc some time during the next sprint.

Reply:
@gjoliver

Reply:
right, I mean what exactly is missing before this conforms to MultiAgentEnv space format?
I actually think comments will work better than docs, since whoever read this line will have my question, and will have no idea why we added this, and there is no way they can find your doc.
but it's not a hard requirement of course. if we leave this as is, you gotta make sure you push through and fix this at some point, otherwise no one know what exactly is left to do.

Reply:
yeah thats p fair. ok lemme do that